feature-utils/poly-look/test/react-components/banners/banner.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { HistoryStub } from "../../utils/history-stub";
+import { Banner } from "../../../src/react-components";
+
+/**
+ * @jest-environment jsdom
+ */
+
+it("Banner renders correctly", () => {
+  const title = "Title";
+  const description = "Description";
+  const { container, getByText, getByRole } = render(
+    <Banner title={title} description={description} icon="icon.svg" />
+  );
+
+  expect(container).toBeTruthy();
+  expect(getByText(title)).toBeTruthy();
```

Comment:
Since `title` is defined above, you can actually check that they are the same. Ditto for description.